src/main/target/link/stm32_flash_split.ld

```diff
@@ -58,7 +58,7 @@ SECTIONS
     *(.tcm_code*)
     . = ALIGN(4);
     tcm_code_end = .; 
-  } >ITCM_RAM
+  } >ITCM_RAM AT >FLASH1
```

Comment:
That's called explicit "load memory address", it's necessary to tell the linker that data should not be attempted to be loaded to ITCM_RAM directly, but rather loaded into FLASH1, and from there it can be copied to ITCM_RAM at run-time.